`ark-relations` build failure

`main` is failing to build:

```
   Compiling ark-relations v0.3.0
   Compiling penumbra v0.1.0 (/Users/redshiftzero/Documents/penumbra/penumbra)
error[E0599]: no method named `parents` found for struct `SpanRef` in the current scope
   --> /Users/redshiftzero/.cargo/registry/src/github.com-1ecc6299db9ec823/ark-relations-0.3.0/src/r1cs/trace.rs:110:28
    |
110 |         let parents = span.parents();
    |                            ^^^^^^^ help: there is an associated function with a similar name: `parent`

For more information about this error, try `rustc --explain E0599`.
error: could not compile `ark-relations` due to previous error
warning: build failed, waiting for other jobs to finish...
error: build failed
```

`span` in the above error message is coming from `tracing`. The 0.3.0 version of `ark-serialize` uses these versions of `tracing` and `tracing-subscriber`:

```
tracing = { version = "0.1", default-features = false }
tracing-subscriber = { version = "0.2", default-features = false, optional = true }
```

We have [these patches](https://github.com/penumbra-zone/penumbra/blob/main/Cargo.toml#L12-L14) applied in our workspace Cargo.toml which point to tracing* branches getting active commits, I'm guessing that's why this failure is happening?

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -10,8 +10,8 @@ members = [
 ]
 
 [patch.crates-io]
-tracing = { git = "https://github.com/tokio-rs/tracing/", branch = "v0.1.x" }
-tracing-subscriber = { git = "https://github.com/tokio-rs/tracing/", branch = "v0.1.x" }
+tracing = { git = "https://github.com/tokio-rs/tracing/", rev = "6cc6c47354ceeb47da7c95faa41c6d29b71b5f37" }
+tracing-subscriber = { git = "https://github.com/tokio-rs/tracing/", rev = "6cc6c47354ceeb47da7c95faa41c6d29b71b5f37" }
 
 # The "ours" branch is based off of v0.3.0
 ark-ff = { git = "https://github.com/penumbra-zone/algebra", branch = "ours" }

--- a/penumbra/Cargo.toml
+++ b/penumbra/Cargo.toml
@@ -34,5 +34,3 @@ hex = "0.4"
 
 [build-dependencies]
 vergen = "5"
-
-

